plugin/Installer/DatabaseChanges.php

```diff
@@ -482,6 +487,47 @@ private function updateSortByUserDefinedDefault()
 				`sortByUserDefinedDefault` NOT LIKE '%#DESC'");
 	}
 
+	/**
+	 *
+	 */
+
+	public function migrationsDataSimilarEstates()
+	{
+		$prefix = $this->getPrefix();
+		$defaultViewOptions = $this->_pWPDB->get_results("select * from " . $prefix . "options where option_name = 'onoffice-default-view'");
+		if(!empty($defaultViewOptions)){
+			add_option('onoffice-similar-estates-settings-view', '');
+			$old_data = [];
+
+			foreach ($defaultViewOptions as $defaultViewOption) {
```

Comment:
There should be only one row with the detail view (it's a serialized `onOffice\WPlugin\DataView\DataDetailView`). How does this work?